linters/ruby/.rubocop.yml

```diff
@@ -18,6 +18,9 @@ Metrics/LineLength:
 Style/AlignParameters:
   EnforcedStyle: with_fixed_indentation
 
+Style/AlignParameters:
```

Comment:
I believe the cop you want to configure here is `Style/AndOr`

Reply:
Oops. Fixed.